### PR TITLE
feat: implemented modal dialogs and menus for windows

### PR DESCRIPTION
**Pull request by CommonGuy**

### CODE REVIEW

File: Source/SpiderEye.Windows/Interop/KeyMapper.cs [hunk 1]
```diff
@@ -36,6 +36,8 @@ private static Keys MapModifier(ModifierKey modifier)
                     return Keys.Control;
 
                 case ModifierKey.Super: // there is Keys.LWin but it's not a modifier key
+                    return Keys.Control;
```
Comment (latonz) [inline Source/SpiderEye.Windows/Interop/KeyMapper.cs:39]: According to the xml doc of ModifierKey.Super this should be the windows key on windows...

Comment (CommonGuy) [reply Source/SpiderEye.Windows/Interop/KeyMapper.cs]: Fixed

File: Source/SpiderEye.Core/UI/ModifierKey.cs [hunk 2]
```diff
@@ -1,42 +1,18 @@
...
-namespace SpiderEye
+﻿namespace SpiderEye
 {
     /// <summary>
-    /// Describes modifier keys.
+    /// Describes system shortcuts.
     /// </summary>
-    [Flags]
-    public enum ModifierKey
+    public enum SystemShortcut
```
Comment (latonz) [inline Source/SpiderEye.Core/UI/ModifierKey.cs:6]: Aren't these file names switched?

Comment (CommonGuy) [reply Source/SpiderEye.Core/UI/ModifierKey.cs]: Ups, bad case of copy-paste

**Pull request merged**